var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -674,14 +674,24 @@ def cmake_args(self):
         if "+lld" in spec:
             projects.append("lld")
         if "+compiler-rt" in spec:
-            projects.append("compiler-rt")
+            if self.spec.satisfies("@15.0.0:"):
+                runtimes.append("compiler-rt")
+            else:
+                projects.append("compiler-rt")
         if "+libcxx" in spec:
-            projects.append("libcxx")
-            projects.append("libcxxabi")
+            if self.spec.satisfies("@15.0.0:"):
+                runtimes.append("libcxx")
+                runtimes.append("libcxxabi")
+            else:
+                projects.append("libcxx")
+                projects.append("libcxxabi")
```

Comment:
FWIW You could tighten this with `extend()`:

```suggestion
                runtimes.extend(["libcxx", "libcxxabi"])
            else:
                projects.extend(["libcxx", "libcxxabi"])
```